tools/standalone-binary-distro/make_distro_builds.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/zsh
+
+gh auth status;
+if [[ $? != 0 ]]; then
+  exit 1
+fi
+
+SDK_VERSION=$(git describe --exact-match --tags)
+if [[ $? != 0 ]]; then
+  echo "❌ Aborting: HEAD doesn't have a tag!"  
+  exit 2
+elif [[ `git status --porcelain` ]]; then
+  echo "❌ Aborting: Working directory has changes!"
+  exit 3
+elif [ ! -f "Cartfile" ]; then
+	echo "❌ Aborting: \`make_distro_builds.sh\` must be run from the same place with `Cartfile`"
+  exit 4
+fi
+OUT_FILENAME="Datadog-$SDK_VERSION.zip"
+
+# create temporary xcconfig for carthage build
+set -euo pipefail
+xcconfig=$(mktemp /tmp/static.xcconfig.XXXXXX)
+trap 'rm -f "$xcconfig" ;' INT TERM HUP EXIT
+echo "BUILD_LIBRARY_FOR_DISTRIBUTION = YES" >> $xcconfig
+export XCODE_XCCONFIG_FILE="$xcconfig"
+
+# clear existing carthage artifacts
+rm -rf Carthage/
+rm Cartfile.resolved
+
+# fetch 3rd party deps via carthage
+echo "carthage bootstrap with no build..."
+carthage bootstrap --no-build
+
+echo "carthage build..."
+carthage build --platform iOS --use-xcframeworks --no-skip-current
+
+# zip artifacts
+cd Carthage/Build/
+zip --symlinks -r $OUT_FILENAME *.xcframework
+mv $OUT_FILENAME ../../$OUT_FILENAME
+cd ../../
+
+# upload zip
+gh release upload $SDK_VERSION $OUT_FILENAME -R DataDog/dd-sdk-ios
```

Comment:
`-R DataDog/dd-sdk-ios` is redundant - we'll be running this tool from our repo.
```suggestion
gh release upload $SDK_VERSION $OUT_FILENAME
```

Reply:
i'll keep it because if the repo has other remotes, `gh` doesn't tell you which repo it tries when it fails.
do you think it may cause problems?

Reply:
No problem at all 👌, just noting this for the solution simplicity (less code). I don't think we'd ever have a scenario for adding remote (IMO equally possible as changing the repo / org name 😉).

Reply:
to be clear, in my local i have 2 remotes (my fork and upstream)
actually, i added my fork to test this PR and will probably remove it after but that's why i add `-R` in the first place.